vlib/v/checker/tests/struct_implements_interface_more_than_once_err.vv

```diff
@@ -0,0 +1,10 @@
+import v.ast.walker { Visitor }
+import v.ast { Node }
```

Comment:
Please use another interface from a module not from the `v.` ones.
`import hash { Hash }` for example, is much lighter to import, and will serve the same exact purpose.

Reply:
Or `Hash32er`, since it is smaller (it has just one method).

Reply:
Ok, `import io { Writer }` seems more suitable.
It has just one method, and it is public.

Reply:
Something like this for example checks for a lot less time:
```
CPU: 98% 0.11s	Real: 0.12s	Elapsed: 0:00.12	RAM: 34592KB	v vlib/v/checker/tests/struct_implements_interface_more_than_once_err.vv
```

compared to the version here:
```
CPU: 86% 0.31s	Real: 0.40s	Elapsed: 0:00.40	RAM: 91568KB	v vlib/v/checker/tests/struct_implements_interface_more_than_once_err.vv
```

Reply:
<img width="1415" alt="image" src="https://github.com/user-attachments/assets/c16ed37d-86df-4067-98fb-1ba1e5feb09a">

the underline for the `io.Writer` is wrong though